src/dlgprefreplaygain.cpp

```diff
@@ -26,24 +35,31 @@ DlgPrefReplayGain::~DlgPrefReplayGain() {
 }
 
 void DlgPrefReplayGain::loadSettings() {
-    if(config->getValueString(ConfigKey(kConfigKey,"ReplayGainEnabled"))==QString("")) {
-        slotResetToDefaults();
-    } else {
-        int iReplayGainBoost =
-                config->getValueString(ConfigKey(kConfigKey, "InitialReplayGainBoost")).toInt();
-        SliderBoost->setValue(iReplayGainBoost);
-        lcddB->display(iReplayGainBoost);
-
-        bool gainEnabled =
-                config->getValueString(ConfigKey(kConfigKey, "ReplayGainEnabled")).toInt() == 1;
-        EnableGain->setChecked(gainEnabled);
-
-        bool analyserEnabled =
-                config->getValueString(ConfigKey(kConfigKey, "ReplayGainAnalyserEnabled")).toInt();
-        EnableAnalyser->setChecked(analyserEnabled);
-    }
+    int iReplayGainBoost = config->getValueString(
+            ConfigKey(kConfigKey, "InitialReplayGainBoost"), "0").toInt();
+    SliderReplayGainBoost->setValue(iReplayGainBoost);
+    LabelCurrentReplayGainBoost->setText(
+            QString::fromUtf8("%1 dB (\xE2\x8C\x80 %2 dB)").arg(
+                  iReplayGainBoost, iReplayGainBoost - 14)); // e.g "0 dB ("Diameter" -14 dB)"
```

Comment:
Just notice, the Ø is German for average value.
http://en.wikipedia.org/wiki/%C3%98_%28disambiguation%29
"
In German-speaking countries, Ø is also used as a symbol for average value: average in German is Durchschnitt, directly translated as cut-through.
"
What is a nice English or international abbreviation we can use instead?